packages/Breadcrumb/src/index.tsx

```diff
@@ -1,21 +1,47 @@
+import utils from '@blaze-react/utils';
 import React, { FunctionComponent } from "react";
 
-const Breadcrumb: FunctionComponent<any> = ({ children }) => (
-  <ul className="breadcrumb">
-    {children.length ? (
-      children.map(
-        (child: any): JSX.Element => (
-          <li key={Math.random()} className="breadcrumb__item">
-            {child}
-          </li>
+interface IBreadcrumb {
+  utils: {
+    uniqueId: (element: any) => string
+  };
+  children: any
+}
+
+const Breadcrumb: FunctionComponent<IBreadcrumb> = ({
+  utils: {
+    uniqueId
+  },
+  children
+}) => {
+
+  const checkChildren = () => {
+    if (Array.isArray(children)) {
+      return children;
+    }
+    return [children]
+  }
+
+  const childs = checkChildren();
+
+  return (
+    <ul className="breadcrumb">
+      {childs.length ? (
+        childs.map(
+          (child: any): JSX.Element => (
+            <li key={uniqueId(child)} className="breadcrumb__item">
+              {child}
+            </li>
+          )
         )
-      )
-    ) : (
-      <li className="breadcrumb__item">{children}</li>
-    )}
-  </ul>
-);
+      ) : (
+          <li className="breadcrumb__item">{childs}</li>
```

Comment:
If you ever are returning an array do we need this li here? If the array have just one element we will have the map above, is right?